cloud-sql/postgres/knex/app.standard.yaml

```diff
@@ -11,7 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-runtime: nodejs10
+runtime: nodejs17
```

Comment:
Even-numbered Node releases are used as long-term support (LTS) candidates.

Perhaps we should use `node16` here?